src/test/java/teammates/test/cases/browsertests/FeedbackMsqQuestionUiTest.java

```diff
@@ -127,6 +128,31 @@ public void testInputValidation() {
         feedbackEditPage.clickAddMsqOtherOptionCheckboxForNewQuestion();
         assertTrue(feedbackEditPage.isElementSelected("msqOtherOptionFlag--1"));
         feedbackEditPage.clickAddQuestionButton();
+
+        ______TS("Check that Max/Min selectable choices cannot be blank");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("MSQ");
+        feedbackEditPage.fillQuestionTextBoxForNewQuestion("Test question text");
+        feedbackEditPage.fillQuestionDescriptionForNewQuestion("more details");
+        feedbackEditPage.fillMsqOptionForNewQuestion(0, "Choice 1");
+        feedbackEditPage.fillMsqOptionForNewQuestion(1, "Choice 2");
+
+        feedbackEditPage.toggleMsqMaxSelectableChoices(NEW_QUESTION_INDEX);
+        feedbackEditPage.fillMsqMaxSelectableChoices(NEW_QUESTION_INDEX, "");
+        feedbackEditPage.clickAddQuestionButton();
+        feedbackEditPage.isInputElementValid(browser.driver.findElement(By.id("msqMaxSelectableChoices--1")));
```

Comment:
You would want to use `feedbackEditPage#getMsqMaxSelectableChoicesElement)` instead

Reply:
This method returns a checkbox element. I want to target the number field in this test. 

```java
    public WebElement getMsqMaxSelectableChoicesElement(int qnNumber) {
        String id = getMsqMaxSelectableChoicesCheckboxId(qnNumber);

        return browser.driver.findElement(By.id(id));
    }
```
I will have to write two more public methods to retrieve the number field element this way. Isn't the current solution in that respect better?

Reply:
Oh. Sorry I thought that was for the input field. I think it's fine to add it into `InstructorFeedbackEditPage`, there's quite many setters and getters for msq already, wouldn't hurt to add another getter for this input field.

Reply:
Okay. :+1: